setup.py

```diff
@@ -33,16 +47,28 @@
         'Programming Language :: Python :: 3.8',
         'Operating System :: OS Independent',
     ],
-    python_requires="!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*",
-    packages=setuptools.find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+    python_requires='>=3.6.0',
+    packages=find_packages(
+        exclude=["*.tests", "*.tests.*", "tests.*", "tests"]
+    ),
+    include_package_data=True,
+    entry_points={
+        'console_scripts': [
+            'htcb=htcrystalball.main:main',
+            'htcrystalball=htcrystalball.main:main'
+        ],
+    },
     install_requires=[
-        'rich',
-        'htcondor'
+        'rich>=6.1.1',
+        'htcondor>=8.8.6',
     ],
     tests_require=[
-        'pytest',
+        'pytest>=6.0.1',
     ],
-    data_files=[
-            ('share/man/man1', ['man/man1/htcrystalball.1', 'man/man1/fetch_condor_slots.1'])
```

Comment:
Do the man pages still install correctly with this removed?

Reply:
Resolving, as a result of the `MAINFEST.in` conversation above.

Reply:
It should, through the manifest file. I'm not sure if this was ever tested through the setup, because the package has always been cloned, but not installed (correct me if I'm wrong). I'll check this out.